assets/src/blocks/Columns/ColumnsTasks.js

```diff
@@ -43,19 +43,19 @@ export const ColumnsTasks = ({ isCampaign, columns, no_of_columns }) => (
                 </h5>
               }
               {description &&
-                <p dangerouslySetInnerHTML={{ __html: description }} />
+                <p dangerouslySetInnerHTML={{__html: description}} />
               }
               {hasImage &&
-                <img src={attachment} alt='' loading='lazy' />
+                <img src={attachment} alt="" loading="lazy" />
               }
               {cta_text && cta_link &&
                 <a
                   className={`btn btn-small btn-${isCampaign ? 'primary' : 'secondary'}`}
                   href={cta_link}
                   target={link_new_tab ? '_blank' : ''}
-                  data-ga-category='Columns Block'
-                  data-ga-action='Call to Action'
-                  data-ga-label={cta_link}
+                  data-ga-category="Columns Block"
+                  data-ga-action="Call to Action"
+                  data-ga-label={cta_link} rel="noreferrer"
```

Comment:
From a quick research, it seems that we shouldn't add `rel="noreferrer"` to internal links (i.e. when `target` is not `_blank`) because it could mess with analytics... So maybe instead of adding it all the time we could use the same condition as for the target? Something like:
```
{...link_new_tab && { rel: 'noreferrer', target: '_blank' } }
```